QUEST_JOBSTEP.tsv

```diff
@@ -2155,14 +2155,14 @@ QUEST_JOBSTEP_20150317_002154	Meet the Cryomancer Master in Klaipeda
 QUEST_JOBSTEP_20150317_002155	The Cryomancer Master says other Masters are teasing him, he wants you to take care of it. Defeat Ironbaum in Siauliai Eastern Woods.
 QUEST_JOBSTEP_20150317_002156	Defeated Ironbaum. Go back to the Cryomancer Master.
 QUEST_JOBSTEP_20150317_002157	Meet the Psychokino Master
-QUEST_JOBSTEP_20150317_002158	Meet the Psychokino Master in Siauliai Miners' Village
-QUEST_JOBSTEP_20150317_002159	Defeat the Werewolf in Siauliai Miners' Village
+QUEST_JOBSTEP_20150317_002158	Meet the Psychokino Master at the Miners' Village
+QUEST_JOBSTEP_20150317_002159	Defeat the Werewolf at the Miners' Village
 QUEST_JOBSTEP_20150317_002160	The Psychokino Master is busy but doesn't want to pass on the chance of defeating the Werewolf to the Linker Master. Defeat the Werewolf in Miners' Village.
 QUEST_JOBSTEP_20150317_002161	Defeated the Werewolf. Go back and report to the Psychokino Master.
 QUEST_JOBSTEP_20150317_002162	Meet the Linker Master
-QUEST_JOBSTEP_20150317_002163	Meet the Linker Master in Siauliai Miners' Village
+QUEST_JOBSTEP_20150317_002163	Meet the Linker Master at the Miners' Village
 QUEST_JOBSTEP_20150317_002164	Get rid of the Werewolf in Siaulai Miner's Village
```

Comment:
Hmm, I thought it was a location change, but it's in the wiki too.  I'll leave them in.